plugins/modules/cluster_shutdown.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: cluster_shutdown
+
+author:
+  - Polona Mihalič (@PolonaM)
+short_description: Shutdown cluster.
+description:
+  - Shutdown cluster.
+version_added: 1.2.0
+extends_documentation_fragment:
+  - scale_computing.hypercore.cluster_instance
+seealso: []
+options:
+  force_shutdown:
+      description:
+        - Bypasses attempts to gracefully shut down all runnning VirDomains before system shutdown.
```

Comment:
This means all running VMs will be force shutdown? The host/node will not be force shutdown?